### DISCUSSION TITLE
How to use latest material icons

### DISCUSSION THREAD

**Discussion by kirschkern** [quasarframework/quasar#11612] [General - Components / Directives / etc] [answered]
I'm having trouble with some material icons used in my quasar project.

Examples:
- When using "vaccinated" I get the icon for "nat"
- When using "handshake" I get no icon at all
- When using "checklist" I get the two icons "check" and "list" instead of the single one

I search for icons to use at https://fonts.google.com/icons?selected=Material+Icons

I use the icons like this: `<q-icon name="checklist" />` or as `<q-btn icon="checklist" />`

What am I missing? Do I have to upgrade/install anything? Any workaround?

Thanks for help and suggestions.
Daniel

**Comment by pdanpdan** [comment, marked as answer]
It works just fine. I suppose they are newer icons, so you need to update `@quasar/extras`

**Reply by kirschkern** [reply]
That solved the issue! Thanks for the fast and great help.

**Reply by hawkeye64** [reply]
@kirschkern You can also check the latest at https://iconexplorer.app
